### DISCUSSION TITLE
Can I use this library for Compose Multiplarform?

### DISCUSSION THREAD

**Discussion by Chandra-Sekhar-Bala** [skydoves/Cloudy#47] [General]
I am absolute new to CMP, and wondering if I could use this library for CMP targeting Android, ios, Desktop(optional).

**Comment by l2hyunwoo** [comment]
As I wrote on [this issue](https://github.com/skydoves/Cloudy/issues/53), Cloudy on CMP will be supported in the second half of this year. Thanks for your interest.

**Reply by l2hyunwoo** [reply]
@Chandra-Sekhar-Bala You can use it v0.3.0 🎉
